pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -87,7 +87,7 @@ func NewConfigObserver(
 				SecretLister_:                kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().Secrets().Lister(),
 				ConfigSecretLister_:          kubeInformersForNamespaces.InformersFor(operatorclient.GlobalUserSpecifiedConfigNamespace).Core().V1().Secrets().Lister(),
 				OpenshiftEtcdEndpointsLister: kubeInformersForNamespaces.InformersFor("openshift-etcd").Core().V1().Endpoints().Lister(),
-				ConfigmapLister:              kubeInformersForNamespaces.InformersFor("openshift-etcd").Core().V1().ConfigMaps().Lister(),
+				ConfigmapLister_:             kubeInformersForNamespaces.ConfigMapLister(),
```

Comment:
required by the recent change in the cloud provider observer - https://github.com/openshift/library-go/commit/16d6830d0b80dc2a3315207116d009ed2dd4cebf#diff-9a42c46842caaad30adbbb0b5557cda9